mutmut_config.py

```diff
@@ -0,0 +1,17 @@
+def pre_mutation(context):
+    line = context.current_source_line.strip()
```

Comment:
This looks magic. Does this help mutmut to not mutate log lines?

Reply:
yes, will put a comment in that file